test/src/api/routes/test-series.js

```diff
@@ -0,0 +1,286 @@
+				if (err) { return done(err); }
+				expect(res).to.have.status(404);
+				expect(res.ok).to.be.false;
+				expect(res.body).to.be.an('object');
+				expect(res.body.message).to.equal('Series not found');
+				return done();
+			});
+	 });
+});
+
+/* eslint-disable no-await-in-loop */
+describe('Browse Series', () => {
+	let work;
+	before(async () => {
+		// create a work which is related to 3 series
+		const seriesBBIDs = [];
+		for (let orderingTypeId = 1; orderingTypeId <= 3; orderingTypeId++) {
+			const seriesBBID = getRandomUUID();
+			const seriesAttribs = {
+				bbid: seriesBBID,
+				// Make type id alternate between "Automatic" (1) and "Manual" (2)
+				orderingTypeId: (orderingTypeId % 2) + 1
+			};
+			await createSeries(seriesBBID, seriesAttribs);
+			seriesBBIDs.push(seriesBBID);
+		}
+		work = await createWork();
+
+		// Now create a revision which forms the relationship b/w work and series
+		const editor = await createEditor();
+		const revision = await new Revision({authorId: editor.id})
+			.save(null, {method: 'insert'});
+
+		const relationshipTypeData = {
+			description: 'test descryption',
+			id: 1,
+			label: 'test label',
+			linkPhrase: 'test phrase',
+			reverseLinkPhrase: 'test reverse link phrase',
+			sourceEntityType: 'Series',
+			targetEntityType: 'Work'
+		};
+		await new RelationshipType(relationshipTypeData)
+			.save(null, {method: 'insert'});
+
+		const relationshipsPromise = [];
+		for (const seriesBBID of seriesBBIDs) {
+			const relationshipData = {
+				sourceBbid: seriesBBID,
+				targetBbid: work.get('bbid'),
+				typeId: relationshipTypeData.id
+			};
+			relationshipsPromise.push(
+				new Relationship(relationshipData)
+					.save(null, {method: 'insert'})
+			);
+		}
+		const relationships = await Promise.all(relationshipsPromise);
+
+		const workRelationshipSet = await new RelationshipSet()
+			.save(null, {method: 'insert'})
+			.then((model) => model.relationships().attach(relationships).then(() => model));
+
+		work.set('relationshipSetId', workRelationshipSet.id);
+		work.set('revisionId', revision.id);
+		await work.save(null, {method: 'update'});
+	});
+	after(truncateEntities);
+
+
+	it('should throw an error if trying to browse more than one entity', (done) => {
+		chai.request(app)
+			.get(`/series?work=${work.get('bbid')}&edition=${work.get('bbid')}`)
+			.end(function (err, res) {
+				if (err) { return done(err); }
+				expect(res).to.have.status(400);
+				return done();
+			});
+	});
+
+	it('should return list of series associated with the work (without any filter)', async () => {
+		const res = await chai.request(app).get(`/series?work=${work.get('bbid')}`);
+		await browseSeriesBasicTests(res);
+		expect(res.body.series.length).to.equal(3);
+	});
+
+	it('should return list of series associated with the work (with Type filter)', async () => {
+		const res = await chai.request(app).get(`/series?work=${work.get('bbid')}&seriesOrderingType=Automatic`);
```

Comment:
With thte change on the other PR:
```suggestion
		const res = await chai.request(app).get(`/series?work=${work.get('bbid')}&orderingType=Automatic`);
```